keyboards/handwired/dactyl_cc/readme.md

```diff
@@ -0,0 +1,51 @@
+# Dactyl-CC / Ergo S-1
+
+![DactylCC](https://i.imgur.com/CUbPLZC.jpeg)  
+*Dactyl-CC*
+
+![ErgoS-1](https://wizardkeyboards.com/wp-content/uploads/2021/11/20211108_233645-1024x498.jpg)  
```

Comment:
This needs to be rehosted.

Reply:
re-hosted as in [ErgoS-1] image URL to move from wizardkeyboards.com to an open platform such as imgur?

Reply:
Changed domain of Ego S-1 image, does it resolve this conversastion?

Reply:
image domain, for Ergo S-1 image, changed to imgur